x-pack/plugin/core/template-resources/src/main/resources/ecs-dynamic-mappings.json

```diff
@@ -114,7 +114,9 @@
               "*.executable",
               "*.name",
               "*.working_directory",
-              "*.full_name"
+              "*.full_name",
```

Comment:
A nit and somewhat unrelated to this PR: you could make the dynamic template a bit more compact by merging `ecs_path_match_keyword_and_match_only_text` and `ecs_match_keyword_and_match_only_text` because both are doing a `path_match` now, after https://github.com/elastic/elasticsearch/pull/96712.